src/main/java/mate/academy/bookstorev2/dto/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.bookstorev2.dto;
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.Getter;
+import lombok.Setter;
+import mate.academy.bookstorev2.validation.FieldMatch;
+import mate.academy.bookstorev2.validation.UniqueEmail;
+
+@Getter
+@Setter
+@FieldMatch(firstString = "password",
+        seconfString = "repeatPassword",
+        message = "The password fields must match")
+public class UserRegistrationRequestDto {
+    @NotBlank
+    @UniqueEmail
```

Comment:
```suggestion
```

If you would like to add custom annotation, use some pattern to check valid email